src/components/TimeseriesChart/stories/basic.story.mdx

```diff
@@ -13,6 +14,7 @@ import {
   series,
   TimeseriesIdsComponentProps,
   SeriesComponentProps,
+  customYdomainSeries,
```

Comment:
`customYdomainSeries` -> `customSeries`

Reply:
This couldn't be done.  StoryBook throws an error because it conflicts with a story named `Custom Series`